MaterialLib/FractureModels/LogPenalty.h

```diff
@@ -0,0 +1,69 @@
+/**
+ * \file
+ *
+ * \copyright
+ * Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "LinearElasticIsotropic.h"
+
+namespace MaterialLib
+{
+namespace Fracture
+{
+/// A penalty function for negative aperture supression used as a multiplier to
+/// the normal fracture stiffness.
+///
+/// The derivative is continuous at aperture = aperture0 and aperture =
+/// aperture_cutoff.
+inline double logPenaltyDerivative(double const aperture0,
+                                   double const aperture,
+                                   double const aperture_cutoff)
+{
+    if (aperture >= aperture0)
```

Comment:
Is it good to compare two real numbers with  ``>=``?

Reply:
I don't see a problem; Comparing with `if (!(aperture < aperuture0))` yields the same assembly using the [ucomisd](http://www.felixcloutier.com/x86/UCOMISD.html) instruction.

At the boundary `aperture == aperture0` the functions result is the same, but the computation (`return 1`) is way easier.

Reply:
I see. Thanks for the explanation.